src/compute-client/src/command.rs

```diff
@@ -162,8 +174,12 @@ impl RustType<ProtoComputeCommand> for ComputeCommand<mz_repr::Timestamp> {
             Some(UpdateMaxResultSize(ProtoUpdateMaxResultSize { max_result_size })) => {
                 Ok(ComputeCommand::UpdateMaxResultSize(max_result_size))
             }
-            Some(CreateTimely(proto_comm_config)) => {
-                Ok(ComputeCommand::CreateTimely(proto_comm_config.into_rust()?))
+            Some(CreateTimely(ProtoCreateTimely { comm_config, epoch })) => {
+                let ProtoComputeStartupEpoch { envd, replica } = epoch.unwrap();
+                Ok(ComputeCommand::CreateTimely {
+                    comm_config: comm_config.unwrap().into_rust()?,
+                    epoch: ComputeStartupEpoch::new(envd.into_rust()?, replica.into_rust()?),
```

Comment:
nit: I think the idiomatic way (and sadly more verbose way) is to add the following implementation
```rust 
impl RustType<ProtoComputeStartupEpoch> for ComputeStartupEpoch { /* ... */ }
```
and then call
```rust
epoch: epoch.into_rust()?
```
so that we can `ComputeStartupEpoch ⇔ ProtoComputeStartupEpoch` mapping can be evolved independent of the context where `ComputeStartupEpoch` is used.